docs/source/how_to_guides/optimization/how_to_scale_optimization_problems.rst

```diff
@@ -0,0 +1,167 @@
+the criterion function. If this is the case, the scaling of the optimization problem
+can be improved by simply dividing all parameter vectors by the start parameters.
+
+Advantages
+
+- very simply
+- works with any type of constraints
+
+Disadvantages
+
+- Makes scaling dependent on start values
+- Parameters with zero start value need special treatment
+
+
+
+.. code-block:: python
+
+    def sphere(params):
+        return (params["value"] ** 2).sum()
+
+
+    start_params = pd.DataFrame(data=np.arange(5), columns=["value"])
+    start_params["lower_bound"] = 0
+    start_params["upper_bound"] = 2 * np.arange(5) + 1
+
+    minimize(
+        criterion=sphere_with_noise,
+        params=start_params,
+        algorithm="scipy_lbfgsb",
+        scaling=True,
+        scaling_options={"method": "start_values", "clipping_value": 0.1},
+    )
+
+
+Divide by bounds
+----------------
+
+In many optimization problems one has additional information on bounds of the parameter
+space. Some of these bounds are hard (e.g. probabilities or variances are non negative),
+others are soft and derived from simple considerations (e.g. if a time discount factor
+were smaller than 0.7, we would not observe anyone to pursue a university degree in a
+structural model of educational choices or if an infection probability was higher
+than 20 % for distant contacts the covid pandemic would have been over after a
+month). For parameters than strongly influence the criterion function, the bounds
+stemming from these considerations are typically tighter as for parameters that have
+a small effect on the criterion function.
+
+Thus a natural approach to improve the scaling of the optimization problem is to re-map
+all parameters such that the bounds [0, 1] for all parameters. This has the additional
+advantage that absolute and relative convergence criteria on parameter changes become
+the same.
+
+Advantages
+
+- very simple
+- works well in many practical applications
+- scaling is independent of start value
+- No problems with division by zero
+
+Disadvantages
+
+- Only works if all parameters have bounds
+- This prohibits some kinds of other constraints in estimagic
+
+
+.. code-block:: python
+
+    def sphere(params):
+        return (params["value"] ** 2).sum()
+
+
+    start_params = pd.DataFrame(data=np.arange(5), columns=["value"])
+    start_params["lower_bound"] = 0
+    start_params["upper_bound"] = 2 * np.arange(5) + 1
+
+    minimize(
+        criterion=sphere_with_noise,
+        params=start_params,
+        algorithm="scipy_lbfgsb",
+        scaling=True,
+        scaling_options={"method": "bounds", clipping_value: 0.0},
+    )
+
+
+Divide by gradient
+------------------
+
+Dividing all parameters by the gradient of the criterion function at the start values
+means that around the start values the problem is scaled optimally. In practice, we do
+not take the exact gradient, but a numerical gradient calculated with a very large step
```

Comment:
Not obvious how this is supposed to be done. Is "large" absolute or relative to start values? What about bounds? Be explicit in relation previous two approaches, I guess you will have to pick one or another for coming up with "correct" step sizes.

Reply:
Explained more precisely now.